CHANGES.rst

```diff
@@ -52,6 +52,7 @@ heasarc
 - Heasarc.locate_data returns empty rows with an error in the error_message column if there are
   no data associated with that row rather than filtering it out. [#3275]
 - Heasarc.locate_data changed to use POST request instead of GET to accomodate large requests. [#3356]
+- maxrec in Heasarc.query_region maxes out on 1e5 regardless of how it is set. Use instead an ADQL query using the TOP parameter set to something larger than 1e5. [#3390]
```

Comment:
no need for the changelog for docs and non user facing code changes, please remove